README.md

```diff
@@ -2,66 +2,50 @@
 
 `main.py` is the entry point for the evaluation process. It takes the following command-line arguments:
 - `--restart`: Path to model restart file.
 - `--mesh-mask`: The name of the mesh mask file.
 - `--output` : The name of the output file where the metrics are stored. The default is `metric_results.txt`.
 
-## Usage [TODO]
-This repo is a WIP and usage is subject to change. Figure 1 below shows how the evaluation procedure works in Spinup-Evaluation.
-
-<p align="center">
-<img src="chart.svg" alt="NEMO flow" width="500"/>
-<figcaption>Fig 1. Evaluation flow diagram</figcaption>
-</p>
 
-Spinup-Evaluation was developed to assess the DINO configuration of NEMO, but new metrics can be added to metrics.py to make it compatible with any ocean model. See [Adding New Metrics](#adding-new-metrics) for details.
+## Using Spinup Evaluation
+This repo is a WIP and usage is subject to change. Figure 1 below shows how the evaluation procedure works in Spinup-Evaluation. The toolbox can be used on `restart` file as `grid` files from NEMO. The restart file is a unified version of individual restart files generated by the running NEMO after getting updated with the latest predicted states. The file(s) path is passed to the `main.py` file and it runs all the metrics defined in `metrics.py`. The metrics resulst are stored in the form of a `csv` file and is then compared to the metrics result of the true data. This comparison is done using statistical metrics like MAE, RMSE and z-score that gives an estimate of the deviation of forecasted spinup from true data.
 
-### Running on Saved Restart File
-To evaluate a state obtained from a checkpoint, run Spinup-Evaluation as follows.
+<div align="center">
+  <img src="chart.svg" alt="NEMO flow" width="500"/>
+  <div><strong>Fig 1.</strong> Evaluation flow diagram</div>
+</div>
 
-```sh
-python main.py \
-  --restart <path-to-restart.nc> \
-  --mesh-mask <path-to-mesh_mask.nc> \
-  --output <path-to-output>
-```
 
-### Running on Predictions [TODO]
-To evaluate a new spin-up state obtained using [Spinup-Forecast](https://github.com/m2lines/Spinup-Forecast) do the following:
-
-* `--predictions`: The path to the directory containing the new `pred_[variable].npy` spin-up states from `Spinup-Forecast`.
-    - pred_so.npy
-    - pred_thetao.npy
-    - pred_zos.npy
+The tool can be run using the command-line instructions provided in the `Command line scripts` section.
 
-<!-- * `--mesh-mask` : Path to the `mesh_mask.nc` file. This file contains the grid information for the model.
-* [Optional] The path to a reference spin-up state. This is used to compare the new spin-up state against a known good state. If not provided, the evaluation will only assess the new spin-up state.
-* [Optional] The restart file from the NEMO/DINO model.
-    i.e. `` -->
 
-<!-- > ![alt text](image.png) -->
+## Command line scripts
 
-## Installation
+### Installing Spinup Evaluation
```

Comment:
As previous, keep this named `Installation` and keep at the same level as before. This should appear before `Usage` .